### PR TITLE
Update veriabs.py

### PR DESCRIPTION
**Pull request by priyankadarke**
veriabs.py with the BaseTool2 class. Tested with benchexec.test_tool_info.

### CODE REVIEW

File: benchexec/tools/veriabs.py [hunk 1]
```diff
@@ -5,12 +5,12 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-import benchexec.util as util
+#import benchexec.util as util
```
Comment (PhilippWendler) [inline benchexec/tools/veriabs.py:8]:
```suggestion
```
Please remove unnecessary lines.

File: benchexec/tools/veriabs.py [hunk 2]
```diff
@@ -45,11 +45,12 @@ def program_files(self, executable):
...
-    def cmdline(self, executable, options, tasks, propertyfile, rlimits):
-        if propertyfile:
-            options = options + ["--property-file", propertyfile]
-        return [executable] + options + tasks
+    def cmdline(self, executable, options, task, rlimits):
+        if task.property_file:
+            options += ["--property-file", task.property_file]
+        return [executable] + options + [task.single_input_file]
 
+    '''
```
Comment (PhilippWendler) [inline benchexec/tools/veriabs.py:53]: Same here, please remove obsolete code.

**Review by PhilippWendler — CHANGES_REQUESTED** [review]
Btw: There is no need to open a new PR for more changes, you can always update your existing PR.

**Review by PhilippWendler — APPROVED** [review]

**Comment by priyankadarke** [comment]
> Btw: There is no need to open a new PR for more changes, you can always update your existing PR.
I agree. I am new to git and wasn't able to revert a buggy commit. So I did this to save time and an iteration of changes.

**Pull request merged**